tests/InternalRequest.php

```diff
@@ -216,6 +216,27 @@ public function setScheme($scheme) {
         return $this;
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function getAssetRoot() {
+        return $this->getRoot();
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function setAssetRoot(string $root) {
+        $this->setRoot($root);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function urlDomain($withDomain = true): string {
+        return "/";
```

Comment:
A forward slash doesn't appear to be a valid return value for a method documented to "conditionally get the domain of the request". The return values appear to be based on `$withDomain`:

https://github.com/vanilla/vanilla/blob/2e0109e3acef1d750c1ec926defa1fe69b104e57/library/core/class.request.php#L1697-L1701